src/room_member.cpp

```diff
@@ -175,8 +175,7 @@ void RoomMember::ReceiveLoop() {
 };
 
 void RoomMember::Join(const std::string& nickname, const std::string& server, uint16_t server_port, uint16_t client_port) {
-    // TODO(Subv): Use client_port.
-    RakNet::SocketDescriptor socket;
+    RakNet::SocketDescriptor socket(client_port, 0);
```

Comment:
this is how it appears all over raknet source, I think we should keep as is and process them all together later [we can create an issue for it?]